DOCS-825: Add documentation for app log-in/sign-up, sign-out, password change, improve manage page (#1376)
viamrobotics/docs: docs/manage/_index.md

---
title: "Manage Robots with the Viam app"
linkTitle: "Manage"
weight: 22
type: docs
no_list: true
description: "A guide to getting started with app.viam.com, a web user interface for building and managing robots with Viam."
aliases:
  - "/viam/app.viam.com/"
---

> The [Viam app](https://app.viam.com/) is a web user interface for managing and building robots.

Author: Done, but I changed to `web UI`-- LMK what you think, agree original phrasing is awkward but using interface makes sense to me so many if users understand it?

## Create Account and Log In

To get started on the Viam app, you must log in as an authorized user.
Navigate to [the main page](https://app.viam.com/).

> If you haven't already, toggle to **Sign Up** to create a profile.

Author: Done!

Select **Log In** to log in to the app with your Gmail credentials or your user address and chosen password.

If you forget your password to the app, click **Forgot password** and enter your email address to obtain instructions to reset your password.

## Manage

You do not have to use the app to use Viam's platform to build your robot, but it has several key features to help you get started managing your robots, including configuring robots with more complicated architecture like sub-parts or remotes:

{{< cards >}}
  {{% card link="/manage/configuration" size="small" %}}
  {{% card link="/manage/fleet" size="small" %}}
  {{% card link="/manage/data" size="small" %}}
  {{% card link="/manage/ml" size="small" %}}
  {{% card link="/manage/parts-and-remotes" size="small" %}}
{{< /cards >}}

<br>

> As an alternative to the app, you can manage and control your robots from the command line with our CLI:

Reviewer: I think it's only an alternative to managing with the app not to the app

Author: @npentrel Ah true, I can see how that would be confusing. Updated

> As an alternative to the app, you can manage and control your robots from the command line with our CLI:

Author: ```suggestion
You can also manage and control your robots from the command line with our CLI:
```

---
title: "Manage Robots with the Viam app"
linkTitle: "Manage"
weight: 22
type: docs
no_list: true

> description: "A guide to getting started with app.viam.com, a web user interface for building and managing robots with Viam."

Author: ```suggestion
description: "A guide to getting started with app.viam.com, a cloud-based application for building and managing robots with Viam."
```
